src/frontend/next/src/components/header/Header.js

```diff
@@ -0,0 +1,94 @@
+import styles from '../../styles/Header.module.css';
```

Comment:
Let's do this in Material UI style as CSS in JS instead.

Reply:
Add a `const theme ={}` in the header.js component with all the CSS inside, right?

Reply:
Hi @izhuravlev,
Let's deal with it in the same way we are doing with the other components.

You can check this one as a reference: https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/next/src/components/BackToTopButton.tsx

Reply:
Thank you @PedroFonsecaDEV, I'll look it up

Reply:
Done, except needs to be fixed after making it a TS file